src/rest/orders.rs

```diff
@@ -248,6 +319,13 @@ mod request {
         pub reduce_only: Option<bool>,
```

Comment:
maybe we should insert the `// linear conditional specifics` comment here too for consistency?